Kudu.Services.Web/Default.cshtml

```diff
@@ -1,11 +1,31 @@
 ﻿@using System.IO
+@using System.Reflection
 @using System.Web.Hosting
 @using Kudu.Services.Web
 @using Kudu.Services
 
 @{
     Layout = "~/_Layout.cshtml";
     Page.Title = "Kudu Services";
+
+    string appServiceVersion = GetAppServiceVersion();
+}
+
+@functions {
+    string GetAppServiceVersion()
+    {
+        Assembly assembly;
+        try
+        {
+            assembly = Assembly.Load("Microsoft.Web.Hosting, Version=7.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35");
+            var fileVersionInfo = System.Diagnostics.FileVersionInfo.GetVersionInfo(assembly.Location);
+            return fileVersionInfo.ProductVersion;
+        }
+        catch
+        {
+            return null;
```

Comment:
if we return null. on the page, will it display "null" or empty? maybe should return empty string or "-" instead?